package/doc/sphinx/source/documentation_pages/coordinate_manipulation.rst

```diff
@@ -0,0 +1,107 @@
+.. -*- coding: utf-8 -*-
+
+.. Working with coordinates and things specific to MDA
+   Maybe a basic overview of numpy arrays, but nothing too much
+   Mostly dealing with lib.distances and ways to abuse it!
+
+Coordinate manipulation in MDAnalysis
+=====================================
+
+.. reminder of how cool numpy arrays are
+
+Coordinate, velocity and force data is the meat of molecular simulation and
+these are made directly available as properties of an AtomGroup.  These
+properties will always be available from the AtomGroup, however if the loaded
+trajectory does not have this data then a ``NoDataError`` will be raised.
+
+::
+
+     >>> import MDAnalysis as mda
+     >>> from MDAnalysisTests.datafiles import PSF, DCD
+     >>> ag = u.select_atoms('name CA')
+     >>> ag.positions
+     array([[ 11.66462231,   8.39347267,  -8.98323059],
+            [ 11.41483879,   5.43442154,  -6.51348448],
+          ...
+     >>> vel = ag.velocities
+     NoDataError: Timestep does not contain velocities
+
+These properties return a **copy** of the data for the AtomGroup at the
```

Comment:
Do really return a copy of the arrays? That seems wasteful and slow in terms of system memory. I thought it was a reference that is returned.